script/Arcueid/Day 8/08_00_ARC08_7.txt

```diff
@@ -230,7 +230,7 @@ Going without my glasses causes a lot of problems, sure,
 -- Page 444, Offset 20823.
 -- 「ええ。超能力は突然変異体だけど、それにだって誕生する土壌が必要なの。
 "Yeah.
- It takes a mutation for someone to be born with supernatural abilities,
+ It takes a mutation for someone to be born with psychic powers,
```

Comment:
https://typemoon.fandom.com/wiki/Psychic

Reply:
Irish is correct.

Reply:
This is fine.